testsuite/features/support/cobbler_test.rb

```diff
@@ -34,6 +44,11 @@ def running?
     result
   end
 
+  ##
+  # It returns a list of the names of the systems, profiles, or distros in the database
+  #
+  # Args:
+  #   what: the type of list you want to get.  Valid values are:
```

Comment:
```suggestion
  #   what: The type of list you want to get.  Valid values are:
  #               - systems
  #               - profiles
  #               - distros


```

Reply:
please edit the identation and don't just commit this, editing the suggestion isn't helping me :sweat_smile: